### ISSUE TITLE
Hexonet Dashboard Widget fails to load

### ISSUE THREAD

**Issue by vitor-ao** [centralnicgroup-opensource/rtldev-middleware-whmcs#232]
**Describe the bug**
After seing the message "Widget is currently disabled. Use the first icon for enabling" and pressing the refresh icon, nothing happens. Also noticed that two files failed to load:
```
<script type="text/javascript" src="/admin/\resources\cnic\assets\js\cnic/widget.js"></script>
<link rel="stylesheet" type="text/css" href="/admin/\resources\cnic\assets\css\cnic/widget.css" />
```
Both of the come with **admin** subfolder attached to the URL, which cause them to give 404 error (not found).

**To Reproduce**
1. Go to portal administrator
2. Make sure HEXONET Dashboard Widget is enabled
3. Press the refresh icon.

**Expected behavior**
After pressing refresh, it should display some data.

**Screenshots**

**Context (please complete the following information):**
This was tested on a development platform only
 - Device: Desktop
 - OS: Windows 10 Pro
 - Browser: Firefox 106.0.5
 - WHMCS Version: 8.5.1
 - PHP Version: 7.4.11
 - ISPAPI Registrar Module Version: 16.3.4

**Comment by KaiSchwarz-cnic** [comment]
@vitor-ao 
Thanks for addressing, we'll have a look at this. Still, let me forward that I wonder about Windows being used as Server Software while it isn't supported by WHMCS. Or have you just provided your Desktop OS for reference?

Thanks!

**Comment by vitor-ao** [comment]
Sorry, mentioned wrong. It's an Apache web server via the [Laragon](https://laragon.org/) app that I use for development.

**Comment by KaiSchwarz-cnic** [comment]
Homepage of Laragon is down for an unknown reason. I'll keep you in the loop here.
Looks like this is working fine on other reseller systems. Would it be possible to get access to your system (WHMCS + cPanel / SSH)?

**Comment by vitor-ao** [comment]
Unfortunately, is a local environment, but thanks for the reseller hint. I've uploaded to the production environment (Linux) and the problem doesn't happen and it load the files correctly.

**Comment by KaiSchwarz-cnic** [comment]
https://github.com/centralnicgroup-opensource/rtldev-middleware-whmcs/releases/tag/v16.3.5

solved your issue in 16.3.5 as well.
